sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ClaimsBasedSecurityChannelTest.java

```diff
@@ -122,10 +113,9 @@ public void authorizesSasToken() {
             .thenReturn(Mono.just(accessToken));
 
         // Act
-        StepVerifier.create(cbsChannel.authorize(tokenAudience, scopes))
+        verifyWithTimeout(StepVerifier.create(cbsChannel.authorize(tokenAudience, scopes))
             .expectNext(accessToken.getExpiresAt())
-            .expectComplete()
-            .verify();
```

Comment:
Isn't it easier to just add the timeout here instead of creating a util method for it?